src/settings/MappingProfiles/MappingProfilesTransformationsModal/TransformationsField/TransformationsField.js

```diff
@@ -73,25 +72,25 @@ export const TransformationField = ({
       isEqual={isEqual}
     >
       {({ fields }) => (
-        <MultiColumnList
-          id="mapping-profiles-form-transformations"
-          fields={fields}
-          contentData={prepareData(fields.value)}
-          totalCount={fields.value.length}
-          columnMapping={columnMapping}
-          columnWidths={columnWidths}
-          visibleColumns={visibleColumns}
-          formatter={formatter}
-          autosize={autosize}
-        />
+        <div className={css.mappingProfilesFormTransformations}>
+          <MultiColumnList
+            id="mapping-profiles-form-transformations"
+            fields={fields}
+            contentData={contentData || fields.value}
```

Comment:
As far as I understand fields.value will be applied will be used if search and filter result is empty. Please, could you clarify why do we need to apply fields.value in this case?

Reply:
Form relates on fields.value in every case. If nothing is provided the form will be empty. fields.value is a source of truth for the form. `contentData` is used for now as the approach for the fields form outside the modal.

Reply:
I guess, contentData is used inside of the modal. But I got your idea, thanks for the clarificaiton

Reply:
You are right. Vice verse to what I wrote, field.value is needed for now for the case outside modal. Should be removed after we get rid of it. Will add comment per your suggestion. Thanks.